ripe/atlas/cousteau/source.py

```diff
@@ -73,6 +77,37 @@ def set_type(self, value):
     doc_type = "Defines the type of probe's source."
     type = property(get_type, set_type, doc=doc_type)
 
+    # tags attribute
+    def get_tags(self):
+        """Getter for tags attribute"""
+        return self._tags
+
+    def set_tags(self, value):
+        """Setter for tags attribute"""
+        log = "Sources fields 'tags' should be a dict in the format " \
+              "{ \"include\": [ \"tag1\", \"tag2\", \"tagN\" ], " \
+              "\"exclude\": [ \"tag1\", \"tag2\", \"tagN\" ] }"
+
```

Comment:
I always avoid using backslashes to break strings. There is a better pythonic way to format the above as:
```python
log = (
    'Sources fields 'tags' should be a dict in the format '
    '{ "include": [ "tag1", "tag2", "tagN" ], '
    '"exclude": [ "tag1", "tag2", "tagN" ] }'
)
```